### ISSUE TITLE
The client crashes when there is a protected label variable in a table or page.

### ISSUE THREAD

**Issue by MattiLE** [microsoft/AL#6331]
**Describe the bug**
The client crashes when there is a protected label variable in a table or page.

**To Reproduce**
Steps and to reproduce the behavior:
    1. Create a new AL Project with AL: Go
    2. Create a simple table and a corresponding page
    3. Add a protected label variable 
    4. Press F5 to publish the app and open the new page in the client -> client crashes
![image](https://user-images.githubusercontent.com/48024891/100722650-1f31e380-33c1-11eb-9d63-d08dac8315ec.png)

AL Code to reproduce the issue 
```
table 50100 MyTable
{
    DataClassification = CustomerContent;

    fields
    {
        field(1; Code; Code[10])
        {
            DataClassification = CustomerContent;
        }
        field(2; Description; Text[100])
        {
            DataClassification = CustomerContent;
        }
    }

    keys
    {
        key(PK; Code)
        {
            Clustered = true;
        }
    }

    protected var
        myInt: Label 'Test';
}
`````` 

**Expected behavior**
I expect that I have access to the protected label variable in a page or table extension and that the page can still be opened ;) 

**5. Versions:**
 - AL Language: v6.1.362735
 - Business Central: Version: DE Business Central 17.1 (Platform 17.0.18204.18738 + Application 17.0.16993.0)

**Comment by sergeyol** [comment]
Agree, either labels should be supported as protected variables, or compiler should warn that it is not supported.

**Comment by thloke** [comment]
Closing as this has been fixed in the 18.0 release (and I believe backported to a 17.x CU, but I can't determine which one)
